UselessMine.cpp

```diff
@@ -323,6 +329,27 @@ void UselessMine::Event (bz_EventData *eventData)
                 else if (mine.defused)
                 {
                     dieData->killerID = mine.defuserID;
+                    if (playerID != mine.owner)
```

Comment:
Add new line before this so there's a space between the killer assignment and the if